app/models/concerns/katalyst/tables/collection/pagination.rb

```diff
@@ -57,7 +57,7 @@ def initialize(app)
           def call(collection)
             @collection = @app.call(collection)
             if collection.paginate?
-              @collection.pagination, @collection.items = pagy(@collection.items, collection.paginate_options.dup)
+              @collection.pagination, @collection.items = pagy(@collection.items, **collection.paginate_options)
```

Comment:
I can't remember if I added this `.dup`.
it may have been necessary as pagy was mutating the options you pass into it.

Reply:
I don't think we need the dup anymore because the splat means that the original hash is no longer passed to the pagy method. This ensures that pagy won't modify the options.